docs/logic-apps/authentication.md

```diff
@@ -39,3 +40,30 @@ using (var logicApp = await LogicAppClient.CreateAsync(resourceGroup, logicAppNa
 {	
 }
 ```
+## Using a AccessToken
+
+The service principal used to request a token will need to have at least [`Logic App Contributor`](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#logic-app-contributor) permissions on all your Azure Logic App instances that you want to test.
+
+The main purpose of authenticating using a token is to avoid distributing sensitive service principle details. More details on requesting a token can be found [here](https://docs.microsoft.com/en-us/rest/api/azure/#acquire-an-access-token). As the testing framework leverages the Azure Management API the `resource` should be set to `https://management.azure.com/`.
```

Comment:
How about this:
```suggestion
The main purpose of authenticating using a token is to avoid distributing sensitive service principle details.  As the testing framework uses the Azure Management API, the `resource` should be set to `https://management.azure.com/`.

*More details on requesting a token can be found [here](https://docs.microsoft.com/en-us/rest/api/azure/#acquire-an-access-token).*
```

However, I would suggest explaining a bit more what you mean with resources (etc) so that people who are new to this can follow along

Reply:
done, please check